### ISSUE TITLE
What should we do about scoped storage in android 10?

### ISSUE THREAD

**Issue by ashpieboop** [nextcloud/android#5016]
Following #5014 @tobiasKaminsky @ezaquarii @AndyScherzinger 

### Actual behaviour
- In android 10, scoped storage prevents the Nextcloud app from reading files in external storages
- Note that it's able to detect presence of files in the special media storages, but it fails to create thumbnails

### Expected behaviour
- It shouldn't fail to read/access external files
 
### Steps to reproduce
1. Have some newly created images in media folders
2. Open auto upload tab, thumbnails fail to appear

There are probably other crashes and bugs related to this.

### Environment data
Android version: 10

Device model: emulator

Stock or customized system: stock with play store

Nextcloud app version: latest master (3.10.0 alpha)

Nextcloud server version: latest stable (17.x)

### Access failing logs
```
D/SyncedFoldersActivity: onResume() starting
D/FileActivity: Operations service connected
E/BitmapFactory: Unable to decode stream: java.io.FileNotFoundException: /storage/emulated/0/DCIM/Camera/IMG_20191216_113726.jpg: open failed: EACCES (Permission denied)
    Unable to decode stream: java.io.FileNotFoundException: /storage/emulated/0/DCIM/Camera/IMG_20191216_113726.jpg: open failed: EACCES (Permission denied)
E/BitmapFactory: Unable to decode stream: java.io.FileNotFoundException: /storage/emulated/0/DCIM/Camera/IMG_20191216_113104.jpg: open failed: EACCES (Permission denied)
[...]
```

Edit: of course, now that the PR is merged, these problems don't happen anymore; but there might be something important to do with scoped storage. Any input/help would be appreciated.

**Comment by AndyScherzinger** [comment]
pinging @tobiasKaminsky @ezaquarii for feedback

**Comment by szaimen** [comment]
Will this become a problem when scoped storage will be mandatory on Android 11?
https://developer.android.com/preview/privacy/storage

**Comment by ArisuOngaku** [comment]
@szaimen yes

There has already been some relevant discussion about it in https://github.com/nextcloud/android/issues/6150#issuecomment-646132122

**Comment by tobiasKaminsky** [comment]
Further readings: 
https://developer.android.com/preview/privacy/scoped-storage
https://developer.android.com/preview/privacy/scoped-storage#toggle-behavior-change

**Comment by AlvaroBrey** [comment]
For the record: we've decided to bypass scoped storage altogether, and [request the `MANAGE_EXTERNAL_STORAGE` permission](https://github.com/nextcloud/android/pull/9228) instead. This lets the app work as closely to how it did before android 11 as possible. 

The caveat was that this permission required explicit approval for Google Play review, but we already got that for 3.18.0 RC5.

Closing for now as this is now resolved.
